src/pages/index.vue

```diff
@@ -1,81 +1,223 @@
+<script setup lang="ts">
+import Button from 'primevue/button'
+import InputText from 'primevue/inputtext'
+import Dropdown from 'primevue/dropdown'
+import { getPokemon, getPokemonByNameOrId } from '../api.calls'
+import { type PaginationSizeTypes, type SinglePokemonResponse } from '../models/pokedex/index'
+import { isFav } from '@/helpers'
+import FavoritesView from '@/components/FavoritesView.vue'
+import { useRouter } from 'vue-router'
+import Toast from 'primevue/toast'
+import { useToast } from 'primevue/usetoast'
+
+const toast = useToast()
+const FINAL_POKEMON_ID = 10277
+const router = useRouter()
+const searchQuery = ref('')
+const pokemon = ref<(SinglePokemonResponse | undefined)[]>([])
+const dropdownSizes: PaginationSizeTypes[] = [25, 50, 100, 'All']
+const selectedSize = ref<PaginationSizeTypes>(25)
```

Comment:
I recommend setting it to the first element in the dropdownSizes array so you don't have to change both in the future